client/src/components/Pill/Pill.css

```diff
@@ -0,0 +1,16 @@
+.pill-container {
+  height: 1.061vw;
+  width: 7.071vw;
+  position: absolute;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  font-family: non-serif;
+  bottom: 0%;
+  font-size: 0.707vw;
+  font-family: sans-serif;
+  font-weight: 350;
```

Comment:
For the font family and font weight, can you use one of the CSS variables?

Reply:
Also font weight has to be a unit of 100 (i.e. 100, 200, 300, 400, 500, all the way up to 900)